lib/components/AlbumScreen/track_list_tile.dart

```diff
@@ -205,18 +226,38 @@ class TrackListTile extends StatelessWidget {
               id: parentItem?.id.raw ?? "",
               item: parentItem,
             )));
+          break;
+        case ItemSwipeActions.nothing:
+          break;
       }
 
-      GlobalSnackbar.message(
-        (scaffold) => FinampSettingsHelper.finampSettings.swipeInsertQueueNext
-            ? AppLocalizations.of(scaffold)!.confirmAddToNextUp("track")
-            : AppLocalizations.of(scaffold)!.confirmAddToQueue("track"),
-        isConfirmation: true,
-      );
+      if (followUpAction != ItemSwipeActions.nothing) {
+        GlobalSnackbar.message(
+          (scaffold) => (followUpAction == ItemSwipeActions.addToNextUp)
+              ? AppLocalizations.of(scaffold)!.confirmAddToNextUp("track")
+              : (followUpAction == ItemSwipeActions.playNext)
+                  ? AppLocalizations.of(scaffold)!.confirmPlayNext("track")
+                  : AppLocalizations.of(scaffold)!.confirmAddToQueue("track"),
+          isConfirmation: true,
+        );
+      }
```

Comment:
let's move this into the switch block, or at least build the message string in the switch and then pass it to the snackbar afterwards